src/SFA.DAS.EmployerIncentives.Web/Controllers/PaymentsController.cs

```diff
@@ -81,6 +85,7 @@ public async Task<IActionResult> ListPaymentsForLegalEntity(string accountId, st
             }
 
             submittedApplications = SortApplications(sortOrder, sortField, submittedApplications);
+            submittedApplications = SetEmploymentCheckFeatureToggle(submittedApplications, _webConfiguration.DisplayEmploymentCheckResult);
```

Comment:
Can this not just go in the loop above?  Otherwise we are looping through the applications twice.

Reply:
SortApplications uses LINQ OrderBy/OrderByDesc, so would still have to have another for... loop, I think.

Reply:
Updated

Reply:
I meant move to between lines 70 and 83 where the collection is already being iterated over.

Reply:
Got you, I saw that comment above that loop about it being a temporary fudge and immediately ignored it :) 
That's updated now